vocdoni/bridge-ui
# Fix: Fallback logo

# Short description
Fetching data for token logos currently doesn't work. This PR fixes this by defaulting to a fallback logo for every token. Additionally, an error handling mechanism was added, such that the fallback token is used if there's an error loading a token's logo.

# How can it be tested
Open the staging environment and check that every token shows the fallback logo.

# Tracker ID
https://linear.app/aragon/issue/VOT-68/token-icons-not-found-on-rinkeby
